### PR TITLE
Problem (Fix #326): Missing support for storing more wallets

### LINKED ISSUE #326: Problem: Missing support for storing more wallets
We only support up to 10 wallets at the moment.

To support up to 100 or even 200 wallets, we need a better illustration. For example, we may have a separate wallet list page with pagination, displaying fields like: `Name`, `Address`, `Network`, `Balance`, etc of each wallet. Of course, users are able to switch between wallets as well. 

The existing bottom-left corner wallet selection may just show up to 10 recently opened wallets.

### PR DESCRIPTION
**Pull request by crypto-matto**
螢幕截圖 2021-04-08 下午3 54 17

### CODE REVIEW

File: src/layouts/home/home.tsx [hunk 1]
```diff
@@ -169,22 +169,22 @@ function HomeLayout(props: HomeLayoutProps) {
...
-    };
+    //   await walletService.setCurrentSession(new Session(walletList[e.key]));
+    //   const currentSession = await walletService.retrieveCurrentSession();
+    //   const currentAsset = await walletService.retrieveDefaultWalletAsset(currentSession);
+    //   setSession(currentSession);
+    //   setUserAsset(currentAsset);
+    //   await walletService.syncAll(currentSession);
+    //   await fetchAndSetNewValidators();
+    //   setLoading(false);
+    // };
```
Comment (eddycpc) [inline src/layouts/home/home.tsx:183]: We can totally remove unused code pieces

File: src/layouts/home/home.tsx [hunk 2]
```diff
@@ -228,8 +228,8 @@ function HomeLayout(props: HomeLayoutProps) {
               </Menu.Item>
             );
           }, session)
-        )}
-        {walletList.length < 10 ? (
+        )} */}
+        {walletList.length < 100 ? (
```
Comment (eddycpc) [inline src/layouts/home/home.tsx:232]:
```suggestion
        {walletList.length <= 100 ? (
```

**Review by eddycpc — CHANGES_REQUESTED** [review]

File: src/layouts/home/home.tsx (hunk 2 above)
Comment (eddycpc) [reply src/layouts/home/home.tsx:232]: There was a small off-by-one error here. Now the limit is 100 wallets

**Comment by eddycpc** [comment]
Looks good overall. Good work 🎊

**Comment by crypto-matto** [comment]
Add spinning during wallet switch
螢幕截圖 2021-04-08 下午4 40 02

**Review by eddycpc — APPROVED** [review]

**Pull request merged**